src/github.com/getlantern/autoupdate/_test_app/README.md

```diff
@@ -0,0 +1,128 @@
+# _test_app
+
+This is an example application that prints the version of the running program
+and the executable file's version in a loop. The purpose of this example is to
+demonstrate how to use `github.com/getlantern/autoupdate` and the `equinox`
+backend.
+
+## Instructions
+
+[Sign up](https://equinox.io/user/signup) for [equinox.io][1].
+
+Create a new application using the equinox
+[dashboard](https://equinox.io/dashboard).
+
+Identify the **Account ID**, **Secret Key** and the new application's
+**Application ID**
+
+Download the equinox tool from the dashboard.
+
+We are going to sign our releases, that's why we need a keypair. Create a new
+one using `openssl`:
+
+```sh
+openssl genrsa -out private.pem 2048
+openssl rsa -in private.pem -out public.pem -pubout
+```
+
+Add a new entry for the application inside `config.go` that resembles the
+following map:
+
+```go
+// config.go
+
+var configMap = map[string]*config{
+  // ...
+  // "_test_app" will be used as the internal name of the application.
+	"_test_app": &config{
+		appID:         `ap_ZZZ`,
+		updateChannel: `stable`,
+		publicKey:     []byte("-----BEGIN PUBLIC KEY-----\nABCDEF\n-----END PUBLIC KEY-----\n"),
+	},
+  // ...
+}
+```
+
+Create a `equinox.yaml` file, we're going to store settings for `equinox`.
+
+```yaml
+---
+account_id: id_XXX
+secret_key: key_YYY
+application_id: ap_ZZZ
+channel: stable
+private-key: ./private.pem
+```
+
+Use the `equinox` tool to update a new version, instead of using floating point
+numbers to describe a release, use integers, it will be easier as the next
+greatest number than `n` is always defined by `n + 1` and we'll have no chance
+of hitting a bug derived from comparison of floating point values.
+
+```sh
+equinox release --config equinox.yaml --version=1 main.go
+# ...
+```
+
+Note: I've found my `equinox` tool does not actually cares about the
+`equinox.yaml` file. So I feed these values directly:
```

Comment:
I created [a tool](https://github.com/getlantern/autoupdate/tree/master/releasetool) to replace the equinox cli, I think it would be better if I remove the references to the old equinox cli and document usage of our `releasetool`.